test/ZendeskApi.Client.Tests/ZendeskApiClientTests.cs

```diff
@@ -4,6 +4,7 @@
 using Microsoft.Extensions.Options;
 using Xunit;
 using ZendeskApi.Client.Options;
+#pragma warning disable 618
```

Comment:
Why's this included?

Reply:
Because the old ZendeskApiClient is marked as Obsolete and produces a warning which we don't want as we'll remove it eventually. We still want to keep the tests.